### DISCUSSION TITLE
How to Update to v20220225?

### DISCUSSION THREAD

**Discussion by digixmax** [rern/rAudio#824] [General]
Just want to confirm that update to v20220225 entails flashing the new version image, rather than using the Update Add-on as was the case with the previous release?

If so, restoring the backup configuration used with the previous release afterward should work?

TIA.

**Comment by ceejayemm** [comment]
There is no reason why you cannot update to 20220225 as normal via the Addons / Update page. Unless you have tried and got some sort of error you are not reporting here ? I have already done this on a couple of my systems without any problems.

Chris

**Reply by digixmax** [reply]
In place of the "Update" button on the Add-On, I found some verbiage saying downloading the image to update.

So, I went ahead and flashed the new image then restored the backup configuration -- no issue so far.

**Comment by ceejayemm** [comment]
Something along the lines of #620 ?

Chris

**Reply by digixmax** [reply]
Yes, although I had used the Add-On update feature to update to the previous version.

Another oddity -- the Add-On now showing the new version number as 20220225 even though the image file is rAudio-1-64bit-20220214.img which I downloaded a couple of weeks ago.

**Comment by ceejayemm** [comment]
The image files are the base on which future dated updates are applied. So no problem there. You only need to reflash a card with a new image if a new major release is made available OR the card gets trashed for some reason.

**Comment by Dave-Kay** [comment]
excuse my little knowledge. i installeds it long ago, configured and and since then i have been using it in the car.
Also i am using  a piusv+ and a allo digione board with this
today i decided, maybe it is a good day to update it for the second time, but it seems i would have to use a new image.... tjat's bad, because i need to configure piusv again, which i can't the manual to do it again and because of 100GB of music on the sd.
is there any way, to update it using the sd-cardwhich is in use, without killing my piusv configuration, or the 100GB of music?
